app/client/cypress/e2e/Regression/ClientSide/ActionExecution/ActionSelector_CopyToClipboardFunctions_spec.ts

```
@@ -0,0 +1,759 @@
+    //     propPane._actionSelectorFieldByLabel("Text to be copied to clipboard"),
+    //     "line1{enter}line2{enter}line3",
+    //     { parseSpecialCharSeq: true },
+    //   );
+    //   propPane.ToggleJSMode("onClick");
+    //   propPane.ValidatePropertyFieldValue(
+    //     "onClick",
+    //     `{{copyToClipboard('aline1\\nline2\\nline3');}}`,
+    //   );
+    // });
+
+    it.only("1. Verify that JSON data can be copied to the clipboard and recognized properly.", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)